src/components/Disputes/DisputeInfo.js

```diff
@@ -121,7 +121,13 @@ const DisputeInfo = React.memo(function({
               </Row>
               <Row compactMode={compactMode}>
                 {disputedActionRadspec ? (
-                  <Field label="Description" value={disputedActionRadspec} />
+                  <Field
+                    label="Description"
+                    value={disputedActionRadspec}
+                    css={`
+                      overflow-wrap: anywhere;
```

Comment:
`overflow-wrap: break-word` strangely doesn't break long words. 

This is a not so common scenario when the long word corresponds to an `appId`
![Screen Shot 2020-07-23 at 12 06 58 AM](https://user-images.githubusercontent.com/22663232/88248918-26115280-cc79-11ea-8e41-a7ac5ffa234b.png)

We could also add the `word-break: break-word` style since `overflow-wrap: anywhere` is [not supported by some browsers](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#Browser_compatibility)

Reply:
Sounds good, let's add it as well and adjust based on how it looks with real usage!